app/main.py

```diff
@@ -1,3 +1,57 @@
+from __future__ import annotations
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    @staticmethod
+    def is_distance(some_data: Distance | int | float) -> Any:
```

Comment:
The name of method looks like checking if `is`, maybe we can do a better naming, also it returns not `Any`, only int or float looks like